src/main/scala/controllers/ApplicationController.scala

```diff
@@ -38,44 +38,32 @@ class ApplicationController @Inject()(actionHandler: ActionHandler, applications
   def editSectionForm(id: ApplicationId, sectionNumber: Int) = Action.async { request =>
     actionHandler.gatherSectionDetails(id, sectionNumber).flatMap {
       case Some((app, af, afs, opp)) =>
```

Comment:
I understand why variable names tend to get shorter, however "af" could be "application future", "application form" or ...
I'd rather use something more readable.